src/reaper_osara.cpp

```diff
@@ -1718,6 +1718,37 @@ void postSelectMultipleItems(int command) {
 	SetCursorContext(1, nullptr);
 }
 
+void postMoveItemOrEnvelopePoint(int command) {
+	TrackEnvelope* envelope = GetSelectedEnvelope(nullptr);
+	if (!envelope) {
+		int count = CountSelectedMediaItems(0);
+		if(count==0) {
+			outputMessage(translate("no items selected"));
+			return;
+		}
+		// Translators: Reported when moving items{} will be replaced by the
+	// number of items; e.g. "2 items moved".
+		switch(command) {
+			case 40117: {
```

Comment:
Comment action name.

```suggestion
			case 40117: { // Item edit: Move items/envelope points up one track/a bit
```